requirements.txt

```diff
@@ -2,8 +2,8 @@ gunicorn==19.8.1
 
 git+https://github.com/CodeForAfrica/HURUmap.git@master#egg=hurumap
 
-# workaround: setup keeps installing wazimap from PYPI
-git+https://github.com/CodeForAfricaLabs/wazimap.git@master#egg=wazimap
+# workaround: `pip install` keeps installing wazimap from PYPI, we need the one from Github
+git+https://github.com/OpenUpSA/wazimap.git@2cdc95447c95f20d1ba9bb2e272eef25075e14ff#egg=wazimap
```

Comment:
Isn't wazimap install in HURUmap?

Reply:
By default, `pip` only installs dependencies define in `install_requires` from `PyPI`, resulting in broken wazimap (`1.1.1`) getting installed @DavidLemayian.

To force `pip` to follow `dependency_links`, we'll need to use `pip install --process-dependency-links`. This would have been my preferred choice but this also has been deprecated and will be removed in the future: https://github.com/pypa/pip/issues/4187

Hence, until a new release of wazimap is pushed to `PyPI` I think this is the easiest and safest way to ensure we get the fixed wazimap installed consistently.